fvtest/compilertest/README.md

```diff
@@ -2,42 +2,69 @@ Test Compiler
 =============
 
 This file contains the Eclipse OMR 'Test compiler', a framework for building
-language independent compiler tests for the Testarossa compiler technology. 
+language independent compiler tests for the Testarossa compiler technology.
 
 Building
 --------
 Before building the Test compiler you must make sure to configure the project
-by following the 'Basic configuration and compile' step as outlined in the 
+by following the 'Basic configuration and compile' step as outlined in the
 top-level `omr/README.md` document.
 
 To build the Test compiler, typing `make` in this directory
-(`omr/fvtest/compilertest`) should suffice. A binary called 
+(`omr/fvtest/compilertest`) should suffice. A binary called
 `testjit` will be produced in the top-level `omr` directory.
 
-BUILD_CONFIG defaults to prod. 
+BUILD_CONFIG defaults to prod.
 
 In the case where `guess-platform.sh` is incorrect, you can set the `PLATFORM`
 environment variables to one of the supported ones, listed in
-`compilertest/build/platforms/host.mk`. 
+`compilertest/build/platforms/host.mk`.
 
 
 Running the Test Compiler
 -------------------------
 
-The Test compiler has been written using [Google Test][gtest], and as such, 
-obeys the Google Test command line arguments, that can be seen by running 
-`testjit --help`. 
+The Test compiler has been written using [Google Test][gtest], and as such,
+obeys the Google Test command line arguments, that can be seen by running
+`testjit --help`.
 
-A very useful subset include: 
+A very useful subset include:
 
-* `--gtest_list_tests` Lists the tests
-* `--gtest_filter=` Filters what runs based on a simple regex. 
+* `--gtest_list_tests` Lists the tests.
+* `--gtest_filter=` Filters what runs based on a simple regex.
+* `--gtest_also_run_disabled_tests` Also run disabled test group.
 
 Options can be passed to the Testarossa compiler technology using the
 `TR_Options` environment variable.
 
     $ TR_Options='log=/tmp/testcomplog,tracefull' testjit
 
 
+Contributing to Test Compiler
+-----------------------------
 
-[gtest]: https://github.com/google/googletest/ 
+Test Compiler is using ILInjectors to invoke Testarossa compiler evaluators.
+In this way, Test Compiler can execute tests directly on Testarossa compiler
+functionalities, independent from upper level languages. Each evaluator has
```

Comment:
I think this reads better:

> The Test compiler uses ILInjectors to directly invoke code generator evaluators.  In this way, the Test compiler can exercise compiler functionality independent from a particular language runtime.

Be consistent with capitalization when referring to the "Test compiler".  In some places you have capitalized Compiler.  I'd say call it "Test compiler" throughout (please sweep through this documentation and make consistent).